### ISSUE TITLE
[BUG] Forgot Password > Error Changing Password

### ISSUE THREAD

**Issue by GeTadle** [RocketChat/Rocket.Chat.iOS#1891]
Environment:
Rocket Chat App version 3.0.0 (188)
Device (or simulator) you're running with: iPad 5th gen ios 11.1.2

Pre-Condition:
Reset password email received

Steps:
1. Click the link on the email received
2. Type your new password
3. Tap reset button

Expected Result
Password reset should be successful

Actual Result
An error message "Error changing password" appeared

Please see attached video
https://www.dropbox.com/s/z4swhrf7rlh4pf2/Video%2006-07-2018%2C%207%2046%2025%20AM.mp4?dl=0

**Comment by BlinkTwice** [comment]
I experienced the same issue and assumed it was a bug but found that the user was not 'activated' as I accidentally enabled the manual user approval functionality (which obviously makes a user inactive by default when signing up). All good after after activating the user from the back-end.

**Comment by rafaelks** [comment]
Problem was on the server, getting fixed in version 0.70.

**Comment by edzillion** [comment]
Request to reopen this issue as I am getting it on the latest rocketchat docker container. (how do I find out the version? I am currently locked out but have SSH access)

Issue is exactly what is described above. Console shows this error:

```
Object { 
  stack: "r@https://chat.joincircles.net/a94c62f4d33a956efafd9ee2fe8e37a898d7f808.js?meteor_js_resource=true:1:4116\n_livedata_result@https://chat.joincircles.net/a94c62f4d33a956efafd9ee2fe8e37a898d7f808.js?meteor_js_resource=true:61:17834\nonMessage

  @https://chat.joincircles.net/a94c62f4d33a956efafd9ee2fe8e37a898d7f808.js?meteor_js_resource=true:61:20170\n_launchConnection/this.socket.onmessage/<

  @https://chat.joincircles.net/a94c62f4d33a956efafd9ee2fe8e37a898d7f808.js?meteor_js_resource=true:59:2772\nforEachCallback@https://chat.joincircles.net/a94c62f4d33a956efafd9ee2fe8e37a898d7f808.js?meteor_js_resource=true:59:3967\n_launchConnection/this.socket.onmessage

  @https://chat.joincircles.net/a94c62f4d33a956efafd9ee2fe8e37a898d7f808.js?meteor_js_resource=true:59:2737\nsockjs-0.3.4.js/u</r.prototype.dispatchEvent

  @https://chat.joincircles.net/a94c62f4d33a956efafd9ee2fe8e37a898d7f808.js?meteor_js_resource=true:59:6461\nsockjs-0.3.4.js/u</k.prototype._dispatchMessage

  @https://chat.joincircles.net/a94c62f4d33a956efafd9ee2fe8e37a898d7f808.js?meteor_js_resource=true:59:20106\nsockjs-0.3.4.js/u</k.prototype._didMessage

  @https://chat.joincircles.net/a94c62f4d33a956efafd9ee2fe8e37a898d7f808.js?meteor_js_resource=true:59:21006\nsockjs-0.3.4.js/u</k.websocket/o.ws.onmessage

  @https://chat.joincircles.net/a94c62f4d33a956efafd9ee2fe8e37a898d7f808.js?meteor_js_resource=true:59:23155\n", 
  isClientSafe: true, 
  error: 403, 
  reason: "Token expired", 
  details: undefined, message: "Token expired [403]", errorType: "Meteor.Error" 
}
```

I found a manual way of resetting my password in this gist: https://gist.github.com/nasrulhazim/10da3980c0e096e85a488e5a97add360

But I am inexpeienced with bcrypt. Should I use an online bcrypt generator like this to convert my password into an encrypted string and then add it like so?

`db.getCollection('users').update({_id:"admin"}, { $set: {"services" : { "password" : {"bcrypt" : "<BCRYPT STRING HERE" } } } })`
